### ISSUE TITLE
Kubectl error

### ISSUE THREAD

**Issue by yash-kulkarni2000** [jaredpetersen/kafka-connect-arangodb#14]
ensure CRDs are installed first.

PFA screenshot

<img width="955" alt="kubectl error" src="https://user-images.githubusercontent.com/74292013/194219785-6ff1c608-2b2f-4bf3-984e-a2ef7f7304f0.PNG">

**Comment by jaredpetersen** [comment]
Yeah, this can happen as a race condition with the kustomize setup. You have to try a couple of times in some cases for it to take.

Let me know if that works for you.
